src/flow_checks/RareDestination.cpp

```diff
@@ -24,16 +24,89 @@
 
 /* ***************************************************** */
 
+u_int32_t RareDestination::getDestinationHash(Flow *f) {
+
+  u_int32_t hash = 0;
+  Host *dest = f->get_srv_host();
+
+  if (f->isLocalToLocal()) {
+
+    char buf[64];
+
+    if (!dest->isMulticastHost() && dest->isDHCPHost()) {
```

Comment:
Please add a comment explaining the logic/idea behind this check

Reply:
Other than that I'd suggest using isLocalUnicastHost function instead of isMulticastHost, to exclude broadcast addresses too.